README.md

```diff
@@ -1,3 +1,6 @@
+> *Info:* Check out [axiom-js](https://github.com/axiomhq/axiomjs), our new cross-platform solution with JavaScript SDKs and libraries. `axiom-node` will be deprecated August 1st 2023.
+> 
```

Comment:
```suggestion
> **Note** Check out [axiom-js](https://github.com/axiomhq/axiomjs), our new cross-platform solution with JavaScript SDKs and libraries.
```

Changed my mind, deprecation note looks weird, let's maybe take more time to think about this.